### PR TITLE
Fix/ROE-2433: Fix for apply-using-paper-form page

### PR DESCRIPTION
**Pull request by mwejuli-ch**
### JIRA link

https://companieshouse.atlassian.net/browse/ROE-2433

### Change description

Ensures that transactionId and submissionId in url for use-paper-form work as expected when REDIS_flag is ON/OFF

### Work checklist

- [x] Tests added where applicable
- [x] UI changes meet accessibility criteria

### CODE REVIEW

File: src/controllers/use.paper.contoller.ts [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
 import { Request, Response } from "express";
+import { isActiveFeature } from "../utils/feature.flag";
+import { getUrlWithTransactionIdAndSubmissionId } from "../utils/url";
 
 export const get = (req: Request, res: Response) => {
   const applyWithPaperFormHeading: string = "You'll need to apply using the paper form";
+  const backLinkUrl = isActiveFeature(config.FEATURE_FLAG_ENABLE_REDIS_REMOVAL)
+    ? getUrlWithTransactionIdAndSubmissionId(config.SECURE_REGISTER_FILTER_WITH_PARAMS_URL, req.params[config.ROUTE_PARAM_TRANSACTION_ID], req.params[config.ROUTE_PARAM_SUBMISSION_ID])
+    : config.SECURE_REGISTER_FILTER_URL;
+  console.log(backLinkUrl);
```
Comment (mwestacott) [inline src/controllers/use.paper.contoller.ts:11]: `console.log` statement needs to be removed.

Comment (mwejuli-ch) [reply src/controllers/use.paper.contoller.ts]: removed.

**Review by mwestacott — APPROVED** [review]

**Comment by mwestacott** [comment]
Code changes look fine though.

**Comment by mwejuli-ch** [comment]
> Code changes look fine though.

Failing checks were a result of Sonar timing out, not the PR per se. I re-ran the failed tasks and jobs and all is good now..

**Review by mwestacott — APPROVED** [review]
LGTM worth getting another approval though.

**Comment by lduranteau** [comment]
LGTM except for coverage.
If it stays this low, the next person will have to add tests on something that doesn't concern his changes.

**Comment by mwejuli-ch** [comment]
> LGTM except for coverage. If it stays this low, the next person will have to add tests on something that doesn't concern his changes.

The requirement is to have coverage of at least 80%. This requirement has been met 😄

**Comment by lduranteau** [comment]
> > LGTM except for coverage. If it stays this low, the next person will have to add tests on something that doesn't concern his changes.
> 
> The requirement is to have coverage of at least 80% for each PR. This requirement has been met 😄

so the margin is 20%.
and you alone have taken the margin, which leaves no more margin at all for the others

**Comment by mwejuli-ch** [comment]
> > > LGTM except for coverage. If it stays this low, the next person will have to add tests on something that doesn't concern his changes.
> > 
> > 
> > The requirement is to have coverage of at least 80% for each PR. This requirement has been met 😄
> 
> so the margin is 20%. and you alone have taken the margin, which leaves no more margin at all for the others

I think the threshold for each PR is a minimum of 80% coverage. This PR meets that threshold which is why all checks have passed. There is no issue with test coverage for this PR.

**Comment by lduranteau** [comment]
> > > > > LGTM except for coverage. If it stays this low, the next person will have to add tests on something that doesn't concern his changes.
> > > > 
> > > > 
> > > > The requirement is to have coverage of at least 80% for each PR. This requirement has been met 😄
> > > 
> > > 
> > > so the margin is 20%. and you alone have taken the margin, which leaves no more margin at all for the others
> > 
> > 
> > I think the threshold for each PR is a minimum of 80% coverage. This PR meets that threshold which is why all checks have passed. There is no issue with test coverage for this PR.
> 
> Yes, this is a coverage statistic for the new code added with this PR, so it should be fine.

I thought it was for the entire code

**Review by lduranteau — APPROVED** [review]

**Pull request merged**